pkg/api/addons.go

```diff
@@ -26,7 +26,7 @@ func (cs *ContainerService) setAddonsConfig(isUpgrade bool) {
 	specConfig := cloudSpecConfig.KubernetesSpecConfig
 	omsagentImage := "mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11012019"
 	var workspaceDomain string
-	if cs.Properties.IsAzureStackCloud() {
+	if cs.Properties.IsCustomCloudProfile() {
 		dependenciesLocation := string(cs.Properties.CustomCloudProfile.DependenciesLocation)
 		workspaceDomain = helpers.GetLogAnalyticsWorkspaceDomain(dependenciesLocation)
 		if strings.EqualFold(dependenciesLocation, "china") {
```

Comment:
Please also handle the logic for the LX cloud. I assume  oms agent Image will also be hosted in LX cloud.

Reply:
I don't think we can add any LX-specific logics in a public repo...